README.md

```diff
@@ -56,6 +56,7 @@ If you're on OSX and have [Homebrew](https://brew.sh/) installed, you can simply
 
 ```bash
 brew install postgresql
+/usr/local/opt/postgres/bin/createuser -s postgres
```

Comment:
@joshdick is there a way to do this using `psql`, assuming all users that have Postgres will have `psql` and things might not necessarily get installed to `/usr/local/opt` (if folks have a different Homebrew install path)

Reply:
How about:

```bash
`brew --prefix`/opt/postgres/bin/createuser -s postgres
```

Should be safe if the package was installed with Homebrew since Homebrew would have laid down that binary, and can spit out its own install path.

Reply:
:shipit: